test/deck_test.rb

```diff
@@ -0,0 +1,47 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/card'
+require './lib/deck'
+
+
+class DeckTest < Minitest::Test
+  def setup
+    @card1 = Card.new(:diamond, 'Queen', 12)
+    @card2 = Card.new(:spade, '3', 3)
+    @card3 = Card.new(:heart, 'Ace', 14)
+    @card4 = Card.new(:club, '5', 5)
```

Comment:
I created @card4 in the setup to maintain test file consistency even though it isn't used until the last test.